src/index.d.ts

```diff
@@ -0,0 +1,40 @@
+declare module "react-native-web-jsonschema-form" {
+  type BaseFormData = Record<string, unknown>;
+  type JsonSchema = Record<string, unknown>;
+  type Uichema = Record<string, unknown>;
+  export interface FormEvent {
+    name: string;
+    prevented: boolean;
```

Comment:
To avoid future problems, we should expose `preventDefault` and `isDefaultPrevented` and keep prevented private.

Reply:
I think this typing is incorrect TBH it should be a class and not an `interface` since interface does not have private.  I just need to find that bit on your code.